spec/oas3.config.json

```diff
@@ -1,7 +1,8 @@
 {
    "groupId": "com.equinix",
    "artifactId": "equinix-openapi-metal",
-   "artifactVersion": "1.0.0",
+   "artifactVersion": "0.1.0",
+   "httpUserAgent": "metal-java/0.1.0",
```

Comment:
Presuming these were the only changes needed and the rest of the changes were generated, is there any reason this is still in draft? Looking to make additional improvements?

Reply:
#93 should be merged first, and then this can be rebased and merged

Reply:
#93 has been merged, so this is good to go